loafwallet/src/WalletCoordinator.swift

```diff
@@ -269,11 +272,9 @@ class WalletCoordinator : Subscriber, Trackable {
 
     private func startActivity() {
         UIApplication.shared.isIdleTimerDisabled = true
-        UIApplication.shared.isNetworkActivityIndicatorVisible = true
```

Comment:
@kcw-grunt so we no more want to show the network activity indicator in the status bar?!

Reply:
This is now deprecated by iOS.  The app no longer has control

Reply:
thanks for explanation